tests/functional/cylc-combination-scripts/02-vr-restart.t

```diff
@@ -39,11 +38,16 @@ export WORKFLOW_RUN_DIR="${RUN_DIR}/${WORKFLOW_NAME}"
 sed -i 's@P1Y@P5Y@' flow.cylc
 run_ok "${TEST_NAME_BASE}-runs" cylc vr "${WORKFLOW_NAME}"
 
+# cat "${TEST_NAME_BASE}-runs.stdout" >&2
+
```

Comment:
```suggestion
```

Reply:
$%^&*(! I'd noted to remove this, then forgot again. Mind like 🏅 🐟